src/libs/_tests/runtime-utils.test.js

```diff
@@ -448,3 +571,119 @@ describe('getDisplayList', () => {
     expect(getAnalysesDisplayList(mockBucketAnalyses)).toBe('testA.Rmd, testB.Rmd')
   })
 })
+
+describe('getCostDisplayForTool', () => {
+  it('will get compute cost and compute status for Galaxy app', () => {
+    //Arrange
+    const expectedResult = `Running $0.53/hr`
+    const app = galaxyRunning
+    const currentRuntime = undefined
+    const currentRuntimeTool = undefined
+    const toolLabel = tools.Galaxy.label
+
+    //Act
+    const result = getCostDisplayForTool(app, currentRuntime, currentRuntimeTool, toolLabel)
+
+    //Assert
+    expect(result).toBe(expectedResult)
+  })
+  it('will get compute cost and compute status for Jupyter runtime', () => {
+    //Arrange
```

Comment:
I favor styling these with all caps... and comments should always have a space after the // or /* delimiter

// ARRANGE

// ACT

// ASSERT

this makes secondary comments within each section look secondary, and these will still stand out.

Reply:
@msilva-broad It'd be good to add any styling guidelines to https://docs.google.com/document/d/1cwQ2IckdSBPqVYUGLjk-Z1zq6qwunml4wkavPoDH48s/edit#heading=h.9np93p8k8ga8 so we can review them more easily and enforce them for all :)

Reply:
I personally don't want to have to us all caps for comments. Definitely something to discuss before adding to the style guidelines.

Reply:
great stuff!  This effort was meant to drive discussion.  The footprint is small.  I suggest we ask Nathan to have a follow-up PR to make further tweaks that come out of us discussing this topic at the UIWG.  I wouldn't want to hold up his PR.

Reply:
an yes, @kyuksel once we have agreement in UIWG, we can update the styling guidelines.

Reply:
To clarify, that doc is just a medium I've used to jot down styling guidelines that've come up throughout the past year in PR reviews, mobings, etc., in hopes of solidifying them as part of UIWG so don't worry about adding to it. All of the existing bullets will need buy-ins as well.